lib/assets/languages.yml

```diff
@@ -1876,6 +1876,10 @@
   :display: 'Chinese'
   :iso6391code: "zh"
   :system: 'iso639-2t'
+  :supported:
+    :sms: false
+    :email: true
+    :phone: true
```

Comment:
I know it's just alphabetical, but adding this to the list of supported languages makes it so that `Chinese` is the first primary language:
![image](https://user-images.githubusercontent.com/52747882/134079294-d3b1342d-9ddf-4e4e-bba5-f07689f07151.png)

I don't want to be overly American-centric, but I'm wondering if it makes sense to special case English, since the large majority of cases are going to be using English. Thoughts?

Reply:
I wouldn't be opposed to re-ordering this list so that English is first. @tstrass @sgober what do you think?

Reply:
I agree with this, and we should probably put all the fully supported ones at the top (ones without *)

Reply:
see the other comment I just left about the asterisks but otherwise love it!

Reply:
I like this a lot. Also commented in other thread about the asterisks